### PR TITLE
Add more trame components

### PR DESCRIPTION
**Pull request by jourdain**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jourdain** [comment]
I will maintain them

**Comment by psavery** [comment]
I will maintain them

**Comment by jourdain** [comment]
@conda-forge/help-python those recipes should be trivial since we grab them from PyPI as they are pure Python but I have no clue how to make the CI happy. 

Is it because I put more than 1 recipe? 

Any help or pointer will be greatly appreciated.

**Comment by ocefpaf** [comment]
The CIs is complaining that you did not specify `hatchling`, the build backend, in your recipe. See https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=852580&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2342

**Comment by jourdain** [comment]
Not sure where the `openslide-python >=1.1.1` dependency is coming from which makes the Windows build fail.

**Comment by jourdain** [comment]
```
Building all recipes
Building cyfi,trame-code,trame-formkit,trame-grid-layout,trame-iframe,trame-leaflet,trame-pvui,trame-quasar,trame-tauri,trame-xterm,wholeslidedata,ximage with local,conda-forge
```

not sure where `cyfi`, `wholeslidedata`, `ximage` are coming from...

File: recipes/trame-leaflet/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Kitware/trame-leaflet
+  summary: Leaflet widget for trame
+  license: MIT
+  license_file:
+    - LICENSE
+    - trame_leaflet/LICENSE
```
Comment (ocefpaf) [inline recipes/trame-leaflet/meta.yaml:40]: This is a symlink to the license file above. No need to add it here, please remove.

Comment (jourdain) [reply recipes/trame-leaflet/meta.yaml]: Sorry about that

**Pull request merged**